docs/config/README.md

```diff
@@ -2544,6 +2545,111 @@ style = "bold yellow"
 symbol = "☁️ "
 ```
 
+## OS
+
+The `os` module shows the current operating system.
+OS information is detected via the [os_info](https://lib.rs/crates/os_info) crate.
+
+::: tip
+
+This module is disabled by default.
+To enable it, set `disabled` to `false` in your configuration file.
+
+:::
+
+### Options
+
+| Option     | Default               | Description                |
+| ---------- | --------------------- | -------------------------- |
+| `format`   | `"[$symbol]($style)"` | The format for the module. |
+| `style`    | `"bold white"`        | The style for the module.  |
+| `disabled` | `true`                | Disables the `os` module.  |
+
+<details>
+<summary>
+This module has an advanced configuration option that controls how the operating system symbol is displayed.
+</summary>
+
+| Advanced Option | Default | Description                                            |
+| --------------- | ------- | ------------------------------------------------------ |
+| `symbols`       |         | A table that maps each operating system to its symbol. |
+
+`symbols` allows you to define arbitrary symbols to display for each operating system type.
+Operating system types not defined by your configuration use the default symbols table below.
+See [os_info::Type](https://docs.rs/os_info/latest/os_info/enum.Type.html) for the operating system types currently recognized by this module.
```

Comment:
I would rather list the supported OS-names directly here, and suggest users create feature requests in starship for unsupported operating systems.